tgui/src/interfaces/computer_fabricator.ract

```diff
@@ -47,10 +38,26 @@
 			<td><b>Card Reader:</b>
 			<td><ui-button action='hw_card' params='{"card" : "0"}' state='{{data.hw_card == 0 ? "selected" : null}}'>None</ui-button>
 			<td><ui-button action='hw_card' params='{"card" : "1"}' state='{{data.hw_card == 1 ? "selected" : null}}'>Standard</ui-button>
-		<tr>
-			<td><b>Confirm Order:</b>
-			<td><ui-button action='confirm_order'>CONFIRM</ui-button>
-	</table>
+		</table>
+		
+		{{#if data.devtype != 2}}
+			<table>
+				<tr>
```

Comment:
you're not closing your table rows, this is functional and ok in most browsers, but is still malformed HTML.